### DISCUSSION TITLE
Updation of the badges across all repositories documentation.

### DISCUSSION THREAD

**Discussion by kiranmhm** [EddieHubCommunity/support#479] [General]
Description:
The badges in the documentation of different repositories need to be updated or made dynamic.( Eg: The number of contributors )
The issues are mainly noticed in the Eddiebot repository and the support repository.
Linked Issues: #469 , https://github.com/EddieJaoudeCommunity/EddieBot/issues/358
Linked Images:
[Eddiebot Repository](https://github.com/EddieJaoudeCommunity/EddieBot)
![image](https://user-images.githubusercontent.com/46129076/101081636-532f2380-35d0-11eb-846f-45afcd64dec9.png)
[Support Repository](https://github.com/EddieJaoudeCommunity/support)
![image](https://user-images.githubusercontent.com/62864373/100609266-b94b4a80-3333-11eb-9d3c-8629576a032d.png)

**Comment by eddiejaoude** [comment]
Yes it would be good to have consistent badges and styles across our organisation repos 👍- I think this could be made into an Issue to be actioned by someone.

**Reply by eddiejaoude** [reply]
Issue was created https://github.com/EddieJaoudeCommunity/EddieBot/issues/358
